packages/token-db/src/queues/token-meta-aggregator.ts

```diff
@@ -0,0 +1,53 @@
+import { Logger } from '@l2beat/backend-tools'
+import { Database } from '@l2beat/database'
+import { Redis } from 'ioredis'
+import { buildTokenMetaAggregatorSource } from '../sources/token-meta-aggregator.js'
+import { setupQueueWithProcessor } from '../utils/queue/queue-with-processor.js'
+import { setupQueue } from '../utils/queue/setup-queue.js'
+import { wrapTokenQueue } from '../utils/queue/wrap.js'
+import { TokenPayload } from './payloads.js'
+
+export async function setupTokenMetaAggregatorQueue({
+  db,
+  logger,
+  connection,
+}: {
+  db: Database
+  logger: Logger
+  connection: Redis
+}) {
+  const deps = {
+    connection,
+    logger,
+  }
+  const queue = setupQueue(deps)
+  const queueWithProcessor = setupQueueWithProcessor(deps)
+  const tokenMetaAggregatorInbox = queue<TokenPayload>({
+    name: 'TokenMetaAggregator.Inbox',
+  })
+  const tokenMetaAggregatorQueue = wrapTokenQueue(tokenMetaAggregatorInbox)
+
+  const tokenMetaAggregator = queueWithProcessor<TokenPayload>({
+    name: 'TokenMetaAggregator.Processor',
+    processor: (job) =>
+      buildTokenMetaAggregatorSource({
+        logger,
+        db,
+        queue: tokenMetaAggregatorQueue,
+      })(job.data.tokenId),
+  })
```

Comment:
```suggestion
  const tokenMetaAggregator = queueWithProcessor<TokenPayload>({
    name: 'TokenMetaAggregator.Processor',
    processor: (job) =>
      buildTokenMetaAggregatorSource({
        logger,
        db,
      })(job.data.tokenId),
  })
```

I'll test it tomorrow but:
- A separate inbox is not needed in this case
- separate inbox is not attached nor routed to any source inbox
- queueWithProcessor result (`.queue`) should be returned as a source inbox (below)
- metadata aggregator does not need to propagate events any further (at least for now) - it simply ingests events once independent sources provided basic information (and some meta) about the token